packages/react/src/components/Card/CardToolbar.jsx

```diff
@@ -176,7 +176,8 @@ const CardToolbar = ({
         <OverflowMenu
           flipped={overflowMenuPosition}
           title={mergedI18n.overflowMenuDescription}
-          iconDescription={mergedI18n.overflowMenuDescription}
+          iconDescription={extraActions.iconDescription || mergedI18n.extraActionLabel}
```

Comment:
this causes a double tooltip. one for the button and one for the icon. Can we not use the the overflowMenuDescription translated string?

Reply:
overflowMenuDescription is present in different places and getting used in different place. So I added the extra prop.

Reply:
Regarding double tooltip: It was there like that from long time. If its not a problem then I can remove that.

Reply:
In the old case it was not an issue because the string value was the same for both

Reply:
I have removed it.  as you said it's kind of redundant.